gui/pause/buttons/pause_button.gd

```diff
@@ -53,8 +53,9 @@ func resize():
 	stars.polygon[2].x = size.x - 1
 
 
-func _on_Button_toggled(button_pressed):
-	if button_pressed:
+@warning_ignore("shadowed_variable_base_class")
+func _on_Button_toggled(is_pressed: bool):
```

Comment:
This should probably follow Godot conventions. The passed parameter can also just be safely discarded.
```suggestion
func _on_button_toggled(_button_pressed: bool):
```

Reply:
The parameter's checked in the very next line???

And yeah, I know the name's unconventional. Auto-generated signal functions are just Like That. We might want to do a project-wide update-and-reconnect session, because this sure ain't the only one. (Or leave it, since it's instantly recognizable as A Signal Function.)

Reply:
`is_pressed` is the getter function for the `button_pressed` property in the base Button class. Since this property is updated before the signal is called, the function still works properly even when discarding the signal parameter. Godot probably only passes the parameter in case the signal is connected to a different node, so I don't think we need it here.

Thought leaving the signal names alone is probably a good idea for now.

Reply:
Fair point, but I always prefer to use the most local copy of the data available. Helps keep functions as pure as possible.